lib/handlers/alias.get.js

```diff
@@ -1,51 +1,36 @@
 'use strict';
 
 const HttpOutgoing = require('../classes/http-outgoing');
+const validators = require('../utils/validators');
+const httpError = require('http-errors');
 const Alias = require('../classes/alias');
 const utils = require('../utils/utils');
 
 const handler = async (sink, req, org, type, name, alias, extra) => {
-    if (typeof org !== 'string' || org === '') {
-        throw new TypeError(
-            ':org is a required url parameter and must be a string'
-        );
-    }
-
-    if (typeof type !== 'string' || type === '') {
-        throw new TypeError(
-            ':type is a required url parameter and must be a string'
-        );
-    }
-
-    if (typeof name !== 'string' || name === '') {
-        throw new TypeError(
-            ':name is a required url parameter and must be a string'
-        );
-    }
-
-    if (typeof alias !== 'string' || alias === '') {
-        throw new TypeError(
-            ':alias is a required url parameter and must be a string'
-        );
+    try {
+        validators.alias(alias);
+        validators.extra(extra);
+        validators.name(name);
+        validators.org(org);
+    } catch (error) {
+        throw new httpError(404, 'Not found');
```

Comment:
404 is true if any of the validated arguments is an empty string but if its an invalid set of characters wouldn't that be a different 4xx error?

Reply:
See my comment in: https://github.com/asset-pipe/core/pull/11#discussion_r336848921